packages/relay/tests/lib/eth.spec.ts

```diff
@@ -1939,6 +1941,10 @@ describe('Eth', async function () {
     it('returns correct transaction for existing hash', async function () {
       // mirror node request mocks
       mock.onGet(`contracts/results/${defaultTxHash}`).reply(200, defaultDetailedContractResultByHash);
+      mock.onGet(`accounts/${defaultFromLongZeroAddress}`).reply(200, {
+        from: `${defaultEvmAddress}`
```

Comment:
Can you update `defaultTransaction.from` to use the `defaultEvmAddress` and this, otherwise it's unclear why in the following assertions the comparisons would be true
```suggestion
        from: `${defaultTransaction.from}`
```

Reply:
Same below also